GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -313,6 +313,23 @@ public synchronized final InputStream getStream() throws IOException{
         return stream;
     }
 
+    /**
```

Comment:
You can't use that call unless you are a subclass of GVRAndroidResource - it is protected. If the subclass overrides GVRResourceVolume.openResource to make an application-specific subclass of GVRAndroidResource, that subclass needs a way to provide an open stream. The GVRAndroidResource is responsible for opening the stream so it needs to override openStream to do so. Once it opens the stream, it needs to set the private "stream" variable to the open InputStream object. Short of making a protected function to do so I cannot think of a better way. Making "stream" and "streamStates" protected exposes even more of the implementation to subclasses.

What I think we SHOULD do is to make openStream private and getStream public. That way to subclass doesn't have to provide a stream at all - it can just use its own internal stream. That is an API change that I would prefer not to make before release 3.2